<launch>

    <group ns="mur620/">
        <arg name="joint_states_topic" default="joint_states"/>
        <arg name="ur_prefix_l" default="UR10_l/"/>
        <arg name="ur_prefix_r" default="UR10_r/"/>
        <arg name="tf_prefix" default="mur620"/>

        

        <rosparam file="$(find mur_control)/config/dual_arm_collision_avoidance.yaml" command="load" subst_value="true"/> 

        <node name="dual_arm_collision_avoidance" pkg="mur_control" type="dual_arm_collision_avoidance.py" output="screen">
            <param name="joint_states_topic" value="$(arg joint_states_topic)" />
            <param name="ur_prefix_l" value="$(arg ur_prefix_l)" />
            <param name="ur_prefix_r" value="$(arg ur_prefix_r)" />
            <param name="tf_prefix" value="$(arg tf_prefix)" />
            
        </node>
    </group>

</launch>
